content/js/index.md

```diff
@@ -107,6 +97,63 @@ skygear.config({
 });
 ```
 
+<a name="new-projects"></a>
+## New project with configured SDK
+
```

Comment:
I think better tell the reader they will expect a new scaffolding project using ReactJS, Webpack and Babel 6, with skygear configured, so the app will be ready to go in X minutes